### PR TITLE
Hotfix/icims retry url no implicit conversion on personid

### PR DESCRIPTION
**Pull request by cored**

### CODE REVIEW

**Comment by naush** [comment]
:+1:

File: app/presenters/icims/candidate_retry_imports_presenter.rb [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+        imported_result.error
+      else
+        DEFAULT_ERROR_MESSAGE
+      end
+    end
+
+    private
+
+    attr_reader :imported_result, :candidate
+  end
```
Comment (viola) [inline app/presenters/icims/candidate_retry_imports_presenter.rb]: looks like both :imported_result, :candidate can be nil? how so - sorry just dont understand implemention

**Comment by jasonfellows** [comment]
:+1:

File: app/presenters/icims/candidate_retry_imports_presenter.rb (hunk 1 above)
Comment (cored) [inline app/presenters/icims/candidate_retry_imports_presenter.rb]: @viola yes, they can be nil. But I'm not sure if this is the correct implementation I feel that I'm hiding the actual error which is what I'm investing now after your comment. Thanks ;-)

File: app/presenters/icims/candidate_retry_imports_presenter.rb [hunk 2]
```diff
@@ -0,0 +1,38 @@
+module Icims
+  class CandidateRetryImportsPresenter
+    DEFAULT_ERROR_MESSAGE = 'Check your email for details on errors'
+
+    def initialize(candidate, imported_result)
+      @candidate = candidate
```
Comment (jasonfellows) [inline app/presenters/icims/candidate_retry_imports_presenter.rb]: Would it be better to validate init params here before memory is allocated? Are there cases where we should be able to create an instance of this class when a param is nil?

File: app/presenters/icims/candidate_retry_imports_presenter.rb (hunk 1 above)
Comment (viola) [inline app/presenters/icims/candidate_retry_imports_presenter.rb]: I see, sure thing @cored

File: app/presenters/icims/candidate_retry_imports_presenter.rb (hunk 2 above)
Comment (cored) [inline app/presenters/icims/candidate_retry_imports_presenter.rb]: Yes probably, I feel the probably I got ahead of myself with the presenter.

**Comment by rwalters** [comment]
+1

**Pull request merged**